config/ecs/container-insights/otel-agent-default-config.yaml

```diff
@@ -0,0 +1,46 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:55680
+      http:
+        endpoint: localhost:55681
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
+  awsecscontainermetrics:
+
+processors:
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+  batch/metrics:
+    timeout: 60s
+
+exporters:
+  awsxray:
+  awsemf/application:
+    log_group_name: aws/ecs/containerinsights/{aws.ecs.cluster.name}/application/metrics
```

Comment:
Let's use `{{ClusterName}}` as template variable. By the time data will be here in exporter, customers may update the default resource attribute names. I am changing the logic in the exporter. It will search for `{{aws.ecs.cluster.name}}` in the background if `{{ClusterName}}` is not present.